lib/kitchen/directions/bake_notes/bake_numbered_notes.rb

```diff
@@ -26,6 +29,28 @@ def self.bake_note(note:)
           HTML
         )
       end
+
+      def self.bake_note_exercise(note:)
+        exercise = note.exercises.first
+        exercise.solution ? exercise.add_class('os-hasSolution unnumbered') : exercise.add_class('unnumbered')
```

Comment:
This has behavior that is based on `exercise.solution` being truthy -- this check is duplicated by the `return unless exercise.solution` on line 39.  Also this line always adds the class 'unnumbered' which isn't all that clear right away.  I think it'd be more straightforward to split this into two lines, this first one should be

```ruby
exercise.add_class('unnumbered')
```

to make it clear that this will always happen.  And then after line 39 you can say

```ruby
exercise.add_class('os-hasSolution')
```

Reply:
This is the way it is because the easybaked version has `os-hasSolution` before `unnumbered` in the class ordering, and it shows up in the diff if I don't add these classes in that order. However, now that you say this, I realize that it would probably be better to alphabetize class names in the normalize script instead of making Kitchen deal with it.

Reply:
I've now made this change https://github.com/openstax/recipes/pull/34